Update masterPBR.md

PatrickRyanMS opened the pull request:
Add section about controlling specular reflections on metallic-roughness PBRMaterial. Explain parameters stemming from KHR_material_specular and similarly named parameters already existing in PBRMaterial and how they interact.

closes #883

PatrickRyanMS committed:
Update masterPBR.md

Add section about controlling specular reflections on metallic-roughness PBRMaterial. Explain parameters stemming from KHR_material_specular and similarly named parameters already existing in PBRMaterial and how they interact.

bghgary reviewed (commented):
Looks good, though @sebavan should probably review the specifics.

PatrickRyanMS committed:
Updates for PBRMaterial specular

Added a paragraph explaining F0 and F90 as well as some nits. Added a PG example of how to set up PBRMaterial for specular.

PatrickRyanMS committed:
Update content/features/featuresDeepDive/materials/using/masterPBR.md

Co-authored-by: Gary Hsu <bghgary@users.noreply.github.com>

PatrickRyanMS committed:
Create pbrMaterial_specular.jpg

PatrickRyanMS committed:
Merge branch 'master' of https://github.com/PatrickRyanMS/Documentation
